bidding_auction_event_level_reporting.md

```diff
@@ -240,9 +240,10 @@ Note:
 
 - `buyerReportingMetadata` additionally contains all the metadata present in the
   `sellerReportingMetadata`.
-- The 'recency`, `modelingSignals` and `joinCount` fields are noised using a
-  [noising scheme][10].
+- The `recency`, `modelingSignals` and `joinCount` fields are noised using the noising scheme described [here][14]. The parameters `bid`, `desirability` and `adCost` will be rounded to a floating point number with an 8 bit mantissa and 8 bit exponent Currently the noising can be enabled by setting [ENABLE_REPORT_WIN_INPUT_NOISING][15] to true. However, this flag will be removed and noise will be applied by default in B&A from Beta 2.
```

Comment:
Need a period between `exponent` and `Currently`.

Reply:
Done.